packages/common-common/package-lock.json

```diff
@@ -0,0 +1,4629 @@
+{
```

Comment:
Is there a reason we need to add a package-lock here?

Reply:
Ah I used npm here for some other testing. Can remove